app/components/post_list/post/post.tsx

```diff
@@ -241,7 +240,7 @@ const Post = ({
                 theme={theme}
             />
         );
-    } else if (post.type === 'custom_calls' && callsFeatureEnabled) {
+    } else if (post.type === Posts.POST_TYPES.CUSTOM_CALLS) {
```

Comment:
by 1 I meant a function like `isSystemMessage` or `isPostEphemeral` that basically receives the post as argument and return a boolean.

In this case it would be something like

```
export const isCallPost(post) {
    return post.type === Posts.POST_TYPES.CUSTOM_CALLS;
}
```

and this would live in a utility file of some sort, probably `utils/posts` but no need to add it now.